### PR TITLE
Update american.ruleset

### PR DESCRIPTION
**Pull request by Rorr4**
Added more leaders and more nations to civilwar_nations

### CODE REVIEW

File: data/nation/american.ruleset [hunk 1]
```diff
@@ -17,6 +17,7 @@ legend=_("The United States of America achieved its independence from\
 leaders = {
  "name",                        "sex"
  "George Washington",           "Male"
+ "Martha Washington",    	"Female"
```
Comment (lmoureaux) [inline data/nation/american.ruleset:20]: Can we try to keep the nice alignment intact?

**Review by lmoureaux — COMMENTED** [review]
Thanks for the update! The addition of a few more women to the list is much appreciated! I have one small coding style comment (in the next section) and a few suggestions to be discussed:

In order to increase the proportion of women, we could consider removing some less known presidents. From my European point of view, Hamilton/Monroe/Jackson--Taylor/Andrew Johnson/Cleveland/McKinley could be candidates for deletion.
We could also include a few important female senators or representatives, maybe Jeannette Rankin, Nancy Kassebaum, Kamala Harris...

File: data/nation/american.ruleset (hunk 1 above)
Comment (Rorr4) [reply data/nation/american.ruleset:20]: I would say remove Hamilton as he was never a president.

**Comment by jwrober** [comment]
See #2044

**Pull request closed without merging**